### PR TITLE
Feature/vonk 3891 swagger UI pipeline

### PR DESCRIPTION
**Pull request by eugene-blokhin**

### CODE REVIEW

**Comment by cknaap** [comment]
Focus on swagger-initializer.js
index.css
azure-pipeline-update-swagger.yml

File: azure-pipeline-update-swagger.yml [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    cd $(Build.SourcesDirectory)/firely-docs-firely-server
+
+    if [[ `git status --porcelain` ]]; then
+      git checkout -b $(RemoteBranchName)
+
+      git config --global user.email 'automated-build@azuredevops.com'
+      git config --global user.name 'Azure DevOps build'
+
+      git add .
+      git commit -m 'Updating public documentation'
```
Comment (cknaap) [inline azure-pipeline-update-swagger.yml:81]:
```suggestion
      git commit -m 'Updated OpenAPI docs for <Firely Server version - should be in CapStmt>'
```

File: azure-pipeline-update-swagger.yml [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+      git remote add origin2 https://$(git_token)@github.com/FirelyTeam/firely-docs-firely-server.git
+      git push --set-upstream origin2 '$(RemoteBranchName)'
+
```
Comment (cknaap) [inline azure-pipeline-update-swagger.yml:85]: I found docs on creating a PR using `git request-pull` https://hackernoon.com/how-to-git-pr-from-the-command-line-a5b204a57ab1

**Review by cknaap — CHANGES_REQUESTED** [review]
Impressed by the PS scripting!
I could not judge index.css or swagger-initializer.js very well as I have no clue how Swagger works.
Few things on the pipeline itself though.

For clarity of the PR I would leave the generated stuff (OpenApiJson folder) out of it.

File: azure-pipeline-update-swagger.yml [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+          # The code below converts file names to human-readable resource names. Examples:
+          # OpenApi_R4_AdverseEvent.json =>  "Adverse Event"
+          # OpenApi_R4_Bundle.json =>  "Bundle"
+          # OpenApi_R4__Root.json => "System Operations" (exception)
+          $name = $_.Name -replace '(.*_)+' `
+          -replace '.json' `
+          -replace '^Root$', 'SystemOperations' `
+          -csplit '(?=[A-Z])' -ne '' -join ' '
+      
+          [PSCustomObject]@{
```
[the remark this answers is not in the document]
Comment (eugene-blokhin) [reply azure-pipeline-update-swagger.yml:57]: It is a data type in PowerShell :)

File: azure-pipeline-update-swagger.yml (hunk 2 above)
Comment (eugene-blokhin) [reply azure-pipeline-update-swagger.yml:85]: This command is meant for a different thing :(
https://stackoverflow.com/a/45721044/1523432

**Pull request closed without merging**